frontend/src/app/components/containers/parties/edit/party-edit.tsx

```diff
@@ -0,0 +1,340 @@
+import { FC, useCallback, useMemo } from "react";
+import { useNavigate, useParams } from "react-router-dom";
+import { useForm, useStore } from "@tanstack/react-form";
+import { z } from "zod";
+import { gql } from "graphql-request";
+import { PartyEditHeader } from "./party-edit-header";
+import { CompSelect } from "@components/common/comp-select";
+import { FormField } from "@components/common/form-field";
+import { useAppDispatch } from "@hooks/hooks";
+import { useGraphQLQuery } from "@graphql/hooks/useGraphQLQuery";
+import { useGraphQLMutation } from "@graphql/hooks/useGraphQLMutation";
+import { ToggleError, ToggleSuccess } from "@common/toast";
+import { openModal } from "@store/reducers/app";
+import { CANCEL_CONFIRM } from "@apptypes/modal/modal-types";
+import { PartyCreateInput, PartyUpdateInput } from "@/generated/graphql";
+import { CompInput } from "@/app/components/common/comp-input";
+
+const GET_PARTY = gql`
+  query GetParty($partyIdentifier: String!) {
+    party(partyIdentifier: $partyIdentifier) {
+      __typename
+      partyIdentifier
+      partyTypeCode
+      shortDescription
+      longDescription
+      createdDateTime
+      person {
+        personGuid
+        firstName
+        lastName
+      }
+      business {
+        businessGuid
+        name
+      }
+    }
+  }
+`;
+
+const GET_PARTY_TYPE_CODES = gql`
```

Comment:
I'm not requesting changes, but I am flagging this for some team conversation about how we want to manage code tables for the new schemas. For Agency Codes, Alec extended the existing code-table service in the NatCom backend to query the shared schema.

But, that was partly done because there is existing code that relies on that. I'm not sure if that's the pattern we want for code tables. We can discuss as a team and refactor if we need to later.